### PR TITLE
create schema

### PR DESCRIPTION
**Pull request by tri-et**

### CODE REVIEW

File: server/schema/admin/res.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import {Admin} from '../../models'
+import bcrypt from 'bcrypt'
+import jwt from 'jsonwebtoken'
+const resolvers = {
+  RootQuery: {},
+  RootMutation: {
+    async loginAdmin(_, {input}) {
+      const admin = await Admin.findOne({where: {username: input.username}})
+      if (!admin) {
+        throw new Error('Account not found. Please try again!')
```
Comment (hemvanh) [inline server/schema/admin/res.js]: Account not found. -> Admin Account not found.

File: server/schema/admin/res.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      }
+      const valid = await bcrypt.compare(input.password, admin.password)
+      if (!valid) {
+        throw new Error('Wrong Password ...')
```
Comment (hemvanh) [inline server/schema/admin/res.js]: Wrong Admin Password

File: server/schema/admin/res.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      }
+      return (
```
Comment (hemvanh) [inline server/schema/admin/res.js:16]: abundant  "(" ?

**Review by hemvanh — CHANGES_REQUESTED** [review]
change Error message ...

File: server/schema/admin/res.js (hunk 3 above)
Comment (tri-et) [reply server/schema/admin/res.js:16]: eslint auto for like that...can't remove it

**Pull request merged**